tox.ini

```
@@ -66,6 +66,14 @@ commands =
     pylint --rcfile=pylintrc --output-format=parseable {toxinidir}/src/pymovements {toxinidir}/tests
 
 
+[testenv:mypy]
+deps =
```

please add these two lines
```suggestion
basepython = python3.8
changedir = {toxinidir}
deps =
```

maybe it's also a good idea to use `python3.9` for all `basepython` definitions.